src/PendingCalls/AfterEachCall.php

```diff
@@ -54,7 +54,7 @@ public function __destruct()
         $proxies = $this->proxies;
 
         $afterEachTestCase = ChainableClosure::boundWhen(
-            fn (): bool => is_null($describing) || $this->__describing === $describing,
+            fn (): bool => $describing === [] || in_array(end($describing), $this->__describing, true),
```

Comment:
> `end() advances array's internal pointer to the last element, and returns its value.`

Because this function actually mutates the given value, i rather not use it.